packages/changed-elements-react/src/NamedVersionSelector/useNamedVersionsList.ts

```diff
@@ -147,8 +147,14 @@ export function useNamedVersionsList(args: UseNamedVersionListArgs): UseNamedVer
               ),
               changesetId: currentChangesetId,
               changesetIndex: -1,
-              description: null,
-              createdDateTime: "",
+              description: IModelApp.localization.getLocalizedString(
+                "VersionCompare:versionCompare.noNamedVersions",
+              ),
+              createdDateTime: (new Date()).toLocaleDateString(undefined, {
```

Comment:
I would not use a new Date. If anything, we should either not show a date, or show the date of the changeset